app/views/search/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="row">
+  <div class="col-lg-6 col-md-6 mx-auto">
+    <h1 class="fw-light">
+      Welcome to the HPI Compass Portal<%= ", #{current_user.username}" if user_signed_in? && current_user.username.present? %>!
+    </h1>
+    
+    <%= form_with url: search_path, method: :get do |f| %>
+      <div class="field mb-3">
+          <%= f.label :query, "Search", class: 'form-label' %>
+          <%= f.text_field :query, class: 'form-control', autofocus: true, id: "search" %>
+          <%= f.submit "Search", class: 'btn btn-primary mt-3' %>
+      </div>
+    <% end %>
```

Comment:
```suggestion
    <%= form_with url: search_path, method: :get do |form| %>
      <div class="mb-3">
          <%= form.label :query, "Search", class: 'form-label' %>
          <%= form.text_field :query, class: 'form-control', autofocus: true %>
      </div>
      <div class="mb-3">
          <%= form.submit "Search", class: 'btn btn-primary' %>
      </div>
    <% end %>
```

Reply:
> What is the css-class "field" for? It is not part of Bootstrap.

I think it might have been copypasted from a different file that was autogenerated with this class, e.g. https://github.com/hpi-swt2/compass-portal-red/blob/dev/app/views/devise/sessions/new.html.erb

Should be ok to no have it